app/dashboard/helpers.py

```diff
@@ -264,24 +266,13 @@ def process_bounty_details(bountydetails, url, contract_address, network):
         print(e)
         did_change = True
 
-    # Check if we have any fulfillments.  If so, check if they are accepted.
-    # If there are no fulfillments, accepted is automatically False.
-    # Currently we are only considering the latest fulfillment.  Std bounties supports multiple.
-    fments = fulfillments['fulfillments']
-    accepted = fments[-1].get('accepted') if fments else False
-
-    if fments:
-        fulfiller_address = fments[-1].get('payload', {}).get('fulfiller', {}).get('address', '0x0000000000000000000000000000000000000000')
-        fulfiller_email = fments[-1].get('payload', {}).get('fulfiller', {}).get('email', '')
-        fulfiller_github_username = fments[-1].get('payload', {}).get('fulfiller', {}).get('githubUsername', '')
-        fulfiller_name = fments[-1].get('payload', {}).get('fulfiller', {}).get('name', '')
-        fulfiller_metadata = fments[-1].get('payload', {}).get('metadata', {})
-    else:
-        fulfiller_address = '0x0000000000000000000000000000000000000000'
-        fulfiller_email = ''
-        fulfiller_github_username = ''
-        fulfiller_name = ''
-        fulfiller_metadata = {}
+    fments = fulfillments.get('fulfillments', [])
+    logger.debug('Fulfillment data:')
+    logger.debug(json.dumps(fments))
+
+    fments_accepted = [fment.get('accepted') for fment in fments]
+    # If any of the fulfillments have been accepted, the bounty is now accepted and complete.
+    accepted = True if True in fments_accepted else False
```

Comment:
`any(fments_accepted)` should do this in a more elegant way

Reply:
`any` should work here.  Fixed.